src/aditiGJUN20/MissingNumberInIntArray.java

```diff
@@ -0,0 +1,36 @@
+/*
+Program 1: Find the missing number in integer array of 1 to 10?
+Hint: You have given an integer array which contains numbers from 1 to 10 
+but one number is missing, you need to write a Java program to find that 
+missing number in an array
+*/
+
+package aditiGJUN20;
+
+import java.util.Arrays;
+
+public class MissingNumberInIntArray {
+	void findMissingNumber(int[] intArray) {
+		Arrays.sort(intArray);
+		int arrayValue = intArray[0];
+		for (int index = 0; index < intArray.length; index++) {
+			while (intArray[index] == arrayValue)
+				arrayValue++;
+		}
+		System.out.println("Missing number is: " + arrayValue);
+	}
+
+	public static void main(String[] args) {
+		MissingNumberInIntArray missingNumberInIntArray = new MissingNumberInIntArray();
+		// input- 1
+		int[] inputarray1 = { 3, 4, 2, 7, 5, 6, 8 };
```

Comment:
Number 1 is missing here isn't it ? output shows number 9 is missing. 

Reason : we sort an array and we considered first number as starting point, in array 
int[] inputarray1 = { 3, 4, 2, 7, 5, 6, 8 } , number 2 is on first index, it will never talk about 1.  please correct me if I misinterpreted your logic .

If you want to find missing number starting from lowest number in given array then your logic is correct.

Reply:
Yes Krishna. Here, it's taking array input's lowest number as starting point. 
I'm going to rewrite this code and will push that logic.
Thank you for the feedback.